test/functional/feature_assumeutxo.py

```diff
@@ -148,6 +148,42 @@ def test_invalid_mempool_state(self, dump_output_path):
 
         self.restart_node(2, extra_args=self.extra_args[2])
 
+    def test_invalid_snapshot_wrong_coin_code(self, valid_snapshot_path):
+        self.log.info(
+            "Testing snapshot file with an incorrect coin code: height 364 and coinbase 0"
```

Comment:
```suggestion
            "Testing snapshot file with an incorrect coin height 364 and coinbase 0"
```

Reply:
Thanks @ismaelsadeeq, I've squashed them.
Base height is related to AssumeUTXO snapshot, refers to the block height at which the UTXOset snapshot was taken